shimming-toolbox/shimmingtoolbox/optimizer/lsq_optimizer.py

```diff
@@ -333,11 +357,7 @@ def _get_currents(self, unshimmed_vec, coil_mat, currents_0):
             # scipy minimize expects the return value of the residual function to be ~10^0 to 10^1
             # --> aiming for 1 then optimizing will lower that. We are using an initial guess of 0s so that the
             # regularization on the currents has no affect on the output stability factor.
-            if self.opt_criteria == 'mse':
-                stability_factor = self._initial_guess_mse(self._initial_guess_zeros(), unshimmed_vec,
-                                                           np.zeros_like(coil_mat),
-                                                           factor=1)
-            elif self.opt_criteria == 'grad':
+            if 'mse' in self.opt_criteria:
```

Comment:
This will also match self.opt_criteria == "rmse"

Reply:
fixed in [2ae6977](https://github.com/shimming-toolbox/shimming-toolbox/pull/573/commits/2ae6977d3f96b0472bc2adb6546a6d23edca160d)